docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -226,3 +226,23 @@ components:
     memoryLimit: 1024Mi
     mountSources: true
 ```
+
+### State file
+
+When the command `odo dev` is executed, the state of the command is saved in the file `./.odo/state.json`. 
```

Comment:
As discussed and as you changed it in the other files, the file name should now be `./.odo/devstate.json` ^^

```suggestion
When the command `odo dev` is executed, the state of the command is saved in the file `./.odo/devstate.json`. 
```